sdk/sqlmanagement/Azure.ResourceManager.Sql/tests/SqlManagementClientBase.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Sql.Tests
+{
+    [RunFrequency(RunTestFrequency.Manually)]
```

Comment:
Doesn't this mean the test won't be included in CI?  Seems like we want this test in CI.

Reply:
Let's review the choice across all RPs in scrum tomorrow.

Reply:
Also, feels like the attribute belongs on the specific test class, rather than the base class.